doc/frontend/frontend.md

```diff
@@ -15,21 +15,34 @@
 - activity-navbar :
   > display description room and if a user wrinting in the input
 
+  ![html](../../logos/html.png) [html file](front/activity_bar.html)
+
```

Comment:
the link to the html file doesn't work
it need to go back 2 file to be in  scratchy directory like:
../../front/activity_bar.html
and same for every link

Reply:
done here cfb273a04b630a4296bdac87f4fca0a0723ecbf9

Reply:
:facepalm:  you haven't fixed anything

Reply:
really done here 3c73110b3e18d1aaee21de9766b1c34c4e85b98e

Reply:
V3 for problem on directory link